scipy/linalg/_special_matrices.py

```diff
@@ -1396,3 +1398,339 @@ def convolution_matrix(a, n, mode='full'):
         col0 = az
         row0 = raz[-n:]
     return toeplitz(col0, row0)
+
+
+def laplacian_Nd(
+    shape, bc="N", form="sparse", return_evals=False, dtype="float"
+):
```

Comment:
```suggestion
def laplacian_Nd(
    shape, *, bc="N", form="sparse", return_evals=False, dtype="float"
):
```

New functions should use keyword-only arguments where appropriate.

Reply:
Done